src/main/java/com/dape/api/domain/entity/Ticket.java

```diff
@@ -0,0 +1,106 @@
+package com.dape.api.domain.entity;
+
+import com.dape.api.domain.enums.TicketStatusEnum;
+import com.dape.api.domain.enums.TicketTypeEnum;
+import com.dape.api.domain.enums.converter.TicketStatusEnumConverter;
+import com.dape.api.domain.enums.converter.TicketTypeEnumConverter;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Convert;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+
+import java.math.BigDecimal;
+import java.time.LocalDate;
+
+@Entity
+@Table(name = "TICKET")
+public class Ticket {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "IDT_TICKET", nullable = false)
+    private Long idtTicket;
+
+    @Column(name = "NUM_AMOUNT", nullable = false)
+    private BigDecimal numAmmount;
+
+    @Column(name = "DAT_CREATED", nullable = false)
+    private LocalDate datCreated;
```

Comment:
LocalDate é a melhor opção para esse campo? pensando que o Time é uma parte importante dessa informação

Reply:
![LocalDate LocalTime LocalDateTime and ZonedDateTime](https://github.com/user-attachments/assets/fb84c99d-d904-44b5-81e7-8f8edb4bc950)